platooning_control/include/platoon_control_config.h

```diff
@@ -23,38 +23,35 @@
  */
 struct PlatooningControlPluginConfig
 {
-  double timeHeadway = 6.0;
-  double standStillHeadway = 12.0;
-  double maxAccel = 2.5;
-  double Kp = 0.5;
-  double Kd = -0.5;
-  double Ki = 0.0;
-  double maxValue = 100;
-  double minValue = -100;
-  double dt = 0.1;
-  double adjustmentCap = 10;
-  int cmdTmestamp = 100;
-  double integratorMax = 100;
-  double integratorMin = -100;
-  double Kdd = 4.5;                             //coefficient for smooth steering
-  double wheelBase = 3.09;
-  double lowpassGain = 0.5;
-  double lookaheadRatio = 2.0;
-  double minLookaheadDist = 6.0;
+  double standStillHeadway = 12.0; // Standstill gap between vehicles (m)
+  double maxAccel = 2.5;  // Maximum acceleration absolute value used in controller filters (m/s^2)
+  double Kp = 0.5;  // Proportional weight for PID controller
+  double Kd = -0.5;  // Derivative Weight for PID controller
+  double Ki = 0.0;  // Integral weight for PID controller
+  double maxValue = 100;  // Max value to restrict speed cmd (m/s)
+  double minValue = -100; // Min value to restrict speed cmd (m/s)
+  double dt = 0.1; // Timestep to calculate ctrl commands (s)
+  double adjustmentCap = 10;  // Adjustment cap for speed command (m/s)
+  int cmdTmestamp = 100;  // Timestamp to calculate ctrl commands (ms)
+  double integratorMax = 100; // Max limit for integrator term
+  double integratorMin = -100;  // Max limit for integrator term
+  double Kdd = 4.5; //coefficient for smooth steering
+  double wheelBase = 3.09; //Wheelbase of the vehicle (m)
```

Comment:
the wheelbase is in the config struct because it is used in the calculations. But it is not in the node yaml file because its value is updated by the global wheelbase param, not the node's configurable parameters.